framework/src/org/checkerframework/framework/stub/StubParser.java

```diff
@@ -845,8 +880,11 @@ private void annotateParameters(
             TypeParameter param = typeParameters.get(i);
             AnnotatedTypeVariable paramType = (AnnotatedTypeVariable) typeArguments.get(i);
 
-            annotate(paramType, param.getAnnotations());
-            if (param.getTypeBound() != null && param.getTypeBound().size() == 1) {
+            if (param.getTypeBound() == null || param.getTypeBound().isEmpty()) {
+                // No bound so annotations are both lower and upper bounds
+                annotate(paramType, param.getAnnotations());
+            } else if (param.getTypeBound() != null && param.getTypeBound().size() == 1) {
+                annotate(paramType.getLowerBound(), param.getAnnotations());
                 annotate(paramType.getUpperBound(), param.getTypeBound().get(0));
             }
```

Comment:
I confirmed that there are > 1 bounds when the upperbound is an intersection.  I added a warning and I'm working on implementing intersection support.  (But not for this release.)